app/routes/dump.py

```diff
@@ -51,16 +51,16 @@ def dump_routing(schema, questionnaire_store):
         questionnaire_store.metadata,
     )
     full_routing_path = router.full_routing_path()
-    response = []
 
-    for routing_path in full_routing_path:
-        response.append(
-            {
-                "section_id": routing_path.section_id,
-                "list_item_id": routing_path.list_item_id,
-                "routing_path": list(routing_path),
-            }
-        )
+    response = [
+        {
+            "section_id": routing_path.section_id,
+            "list_item_id": routing_path.list_item_id,
+            "routing_path": routing_path.values,
+        }
+        for routing_path in full_routing_path
+    ]
+
```

Comment:
full_routing_path doesn't need to be instantiated. It can go straight into the list comp as it isn't used elsewhere.